pkgs/by-name/lm/lmstudio/linux.nix

```
@@ -28,7 +31,19 @@ appimageTools.wrapType2 {
     mkdir -p $out/share/applications
     cp -r ${appimageContents}/usr/share/icons $out/share
     install -m 444 -D ${appimageContents}/lm-studio.desktop -t $out/share/applications
+
+    # Rename the main executable from lmstudio to lm-studio
+    mv $out/bin/lmstudio $out/bin/lm-studio
+
     substituteInPlace $out/share/applications/lm-studio.desktop \
-      --replace-fail 'Exec=AppRun --no-sandbox %U' 'Exec=lmstudio'
+      --replace-fail 'Exec=AppRun --no-sandbox %U' 'Exec=lm-studio'
+
+    # lms cli tool
+    install -m 755 ${appimageContents}/resources/app/.webpack/lms $out/bin/
+
+    ${lib.getExe patchelf} --set-interpreter "${stdenv.cc.bintools.dynamicLinker}" \
+    --set-rpath "${stdenv.cc.cc.lib}/lib:${stdenv.cc.cc.lib}/lib64:$out/lib:${
+      lib.makeLibraryPath [ stdenv.cc.cc.lib ]
+    }" $out/bin/lms
```

```suggestion
    patchelf --set-interpreter "${stdenv.cc.bintools.dynamicLinker}" \
    --set-rpath "${lib.getLib stdenv.cc.cc}/lib:${lib.getLib stdenv.cc.cc}/lib64:$out/lib:${
      lib.makeLibraryPath [ (lib.getLib stdenv.cc.cc) ]
    }" $out/bin/lms
```
And remove patchelf from inputs.

(Now that I think about it again, `lib.getExe patchelf` would also be incorrect. patchelf would have needed to go in nativeBuildInputs instead)